benchmarks/src/commonMain/kotlin/com/github/michaelbull/result/BindingBenchmark.kt

```diff
@@ -1,16 +1,17 @@
 package com.github.michaelbull.result
 
-import org.openjdk.jmh.annotations.Benchmark
-import org.openjdk.jmh.annotations.BenchmarkMode
-import org.openjdk.jmh.annotations.Mode
-import org.openjdk.jmh.annotations.OutputTimeUnit
-import org.openjdk.jmh.annotations.Scope
-import org.openjdk.jmh.annotations.State
-import org.openjdk.jmh.infra.Blackhole
-import java.util.concurrent.TimeUnit
+import kotlinx.benchmark.BenchmarkMode
+import kotlinx.benchmark.Mode
+import kotlinx.benchmark.OutputTimeUnit
+import kotlinx.benchmark.BenchmarkTimeUnit
+import kotlinx.benchmark.Benchmark
+import kotlinx.benchmark.Blackhole
+import kotlinx.benchmark.State
+import kotlinx.benchmark.Scope
 
+@State(Scope.Benchmark)
```

Comment:
For some reason, for js benchmark target to pick up any tests it needs the whole class annotated, opened a ticket: https://github.com/Kotlin/kotlinx-benchmark/issues/29